prime-angular-frontend/src/app/shared/components/dialogs/content/error-logger/error-logger.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-dialog-content>
+  <p>URL: {{ this.router.url }}</p>
+  <p>Environment: {{ this.config.environmentName }}</p>
+  <p>Version: {{ this.enviornment.version }}</p>
+  <mat-expansion-panel *ngIf="this.error.stack">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        Details
+      </mat-panel-title>
```

Comment:
I think we need some better messaging to the end user, as per "Need UX messaging and next steps" in the Jira ticket.  Get during demo?